pty.go

```diff
@@ -22,6 +22,9 @@ type PTYCommandType uint8
 const (
 	// PTYCommandTypeResize indicates that the command is a resize request.
 	PTYCommandTypeResize PTYCommandType = iota
+
+	// PTYCommandTypeExec indicates that the command should be executed without PTY.
```

Comment:
Is this to be used as an initial command executed in a remote console scenario? The comment is a little bit confusing, as it states "executed without PTY", but then the constant is of `PTYCommandType`. Can you please elaborate on the use case a bit?

Reply:
Ah... so I should probably introduce a new MessageType and a new type altogether for simple commands with no pty. Ok, I'll look at that later, it's not relevant for the qbee-cli terminal support anyway.